en/docs/learn/configuring-a-jdbc-user-store.md

```diff
@@ -24,9 +24,173 @@ Management Related Tables](../../administer/user-management-related-tables) sect
 In this page, you can find following details related to configuring a
 JDBC user store.
 
-## Properties used in JDBC userstore manager
+##Configuring the internal database as JDBC user store. 
 
-Following are the properties used in JDBC user store manager:
+Add the following configuration to `<IS-HOME>/repository/conf/deployment.toml`.
+
+    ``` toml
+    [user_store]
+    type = "database"
+    ```
+   By default it will use the  `database.shared_db` 
+   configurations in the `         deployment.toml       `  file. As the datasource configuration.
+      
+   If you have a requirement of changing the `database.shared_db` configuration see 
+   [Working With Databases](../../administer/working-with-databases).
+  
+   `database.shared_db` is used to store the both registry and user management related data. If you 
+   are having a requirement of using a datasource specified for the user management you can externalize
+   the user management related tables to another external database and use that database
+   as the user store. Please follow the topic `Configuring an external database as JDBC user store`
+   for the instructions.
+   
+##Configuring an external database as JDBC user store. 
+
+  1. Create a database on a [any supported RDBMS database](../../administer/working-with-databases). 
+    
+  2. Following are the example configuration for each database type.
+        
+??? example "PostgreSQL"
+    
+    1. deployment.toml Configurations.
+        ```
+        [database.user]
+        url = "jdbc:postgresql://localhost:5432/gregdb"
+        username = "regadmin"
```

Comment:
Shall we unify userstore names and match the use case

Reply:
done

Reply:
done